lua/dap/ext/vscode.lua

```diff
@@ -96,8 +126,9 @@ end
 
 local function apply_inputs(config, inputs)
   local result = {}
+  local once = {}
   for key, value in pairs(config) do
-    result[key] = apply_input(inputs, value)
+    result[key] = apply_input(once, inputs, value)
```

Comment:
Doesn't this freeze the value for the entire neovim lifetime?

Reply:
I've been using this implementation for the last few weeks, and it should prompt again the next time you start the debugger, at least that's been my experience. The intention is only to avoid re-entering inputs which are repeated in the json. When I have a chance I can add some tests to validate that behavior.